settings/arm9/source/main.cpp

```diff
@@ -213,7 +214,28 @@ void loadThemeList(const char* directory, std::vector<std::string>& list)
 			themeDir = ent->d_name;
 			if (themeDir == ".." || themeDir == "..." || themeDir == ".") continue;
 
-			list.emplace_back(themeDir);
```

Comment:
and use `list->emplace_back(themeDir);` Then this properly compiles (and seems to work on my end).